### DISCUSSION TITLE
Communication via Emoji's

### DISCUSSION THREAD

**Discussion by doubledh** [Imaginary-Horizons-Productions/prophets-of-the-labyrinth-Archive#333] [Ideas]
Discussion for utilizing emoji's in output, to convey visual feedback more quickly instead of conveying feedback semantically

(elements are already emojized)

**Comment by doubledh** [comment]
Idea: have each archetype associated with an emoji 🧑‍🔬 🗡️ 🛡️ 🩸 🧘  👊 . Then, in the combat dialogue for example, you could display that a chemist-flask (alchemist) was the one doing something or receiving damage, and you wouldn't have to parse the word "alchemist" before realizing that it's not you. Also, you can show the party composition in a single string.

**Reply by ntseng** [reply]
I like this one a lot, especially since delver titles are just the archetype.

**Comment by doubledh** [comment]
Idea: associate each player with a color-block 🟥  or number emoji 1️⃣ (perhaps either chosen or generated from the hexcodes): players can identify themselves and each other quickly, without even remembering which proper archetype the others are. Can be used in conjunction with archetype emoji's in case there's a party of 4 necro- I mean hemomancers

**Reply by ntseng** [reply]
I'd probably want to stay away from color blocks because being constrained to the emoji options might not give us enough color-blind accommodating options.

With numbers I'd probably just want to do join order, but wouldn't be sure what to do with players 11 and 12.

As for player uniqueness: I think discord displayname + archetype should suffice. If players pick the same archetype and nickname themselves to the same displayname I think that's on them.

**Comment by doubledh** [comment]
Idea: associate enemies with respective emoji's like bloodtail hawk is 🐦 ... it's probably less likely that we'll find emojis for each and every one of them, but it will help speed up grokking the combat logs

**Reply by ntseng** [reply]
We do have the option to make custom emoji on discord. Might be cool to use those as part of the "graphics engine", but they might be too small to really show anything cool. The other thing to note on custom emoji: select menu and some areas of message embeds don't parse emoji and mentions, so we wouldn't be able to use them there.

**Comment by doubledh** [comment]
Idea: associate weapons with emojis ⚔️ , such that the associated moves in the combat logs can be quickly grokked. like enemies, there may not be enough or the correct emoji's available to accomplish this. Weapon Emoji's could also be used in select menus or inspect-self 👍

**Reply by ntseng** [reply]
Can't find a :scythe: emoji :(

**Comment by doubledh** [comment]
Idea: associate weapon UPGRADES with emoji's... 0 clue if we can symbolize the abstract concepts of the upgrades (like THICK), but we could have the upgrades show in conjunction with the weapons

**Reply by ntseng** [reply]
Agreed this seems difficult.

**Comment by doubledh** [comment]
Idea: represent damage/effect in combat logs with emoji's (also example of most of the proposed ideas so far)
 🟥 🧑‍🔬 >  2️⃣  🐦 🍏 Chemist poisoned bloodtail hawk 2 !
🟦 🛡️ > 1️⃣ 🦌 🔥  Knight hit elkimist for 50 fire damage !  
 2️⃣ 🐦 > 🟥 🧑‍🔬 💢 ❗ bloodtail hawk 2 critically hit chemist for 20 damage!

**Reply by ntseng** [reply]
I think this is its own kind of incomprehensible. https://www.emojicode.org/
